ow_sim_tests/test/terrain_interaction.py

```diff
@@ -427,13 +464,13 @@ def test_06_deliver(self):
 
```

Comment:
The new assert works great! However, due to unreliable delivery of sample into the sample dock this test should be decorated with `@unittest.expectedFailure` at the very top of it. This is a temporary measure until we solve the delivery issue.